lldb/include/lldb/Host/Host.h

```diff
@@ -31,6 +31,22 @@ class ProcessInstanceInfo;
 class ProcessInstanceInfoMatch;
 typedef std::vector<ProcessInstanceInfo> ProcessInstanceInfoList;
 
+// Always on system log category and channel.
```

Comment:
So is the log naming `system system` or for now is it just `system` and we might add sub-channels later?

Reply:
I think we should have a comment that explains what this channel is supposed to be used for?